creator-node/src/routes/nodeSync.js

```diff
@@ -308,19 +308,21 @@ async function _nodesync (req, walletPublicKeys, creatorNodeEndpoint, dbOnlySync
           transaction
         })
         const fetchedLatestBlockNumber = fetchedCNodeUser.latestBlockNumber
+        const fetchedLatestClockVal = fetchedCNodeUser.clock
 
         // Delete any previously stored data for cnodeUser in reverse table dependency order (cannot be parallelized).
         if (cnodeUser) {
           // Ensure imported data has higher blocknumber than already stored.
-          // TODO - replace this check with a clock check (!!!)
           const latestBlockNumber = cnodeUser.latestBlockNumber
+          const latestClockValue = cnodeUser.clock
 
           if (!dbOnlySync) {
             if ((fetchedLatestBlockNumber === -1 && latestBlockNumber !== -1) ||
```

Comment:
do we need this blocknumber condition at all anymore? could we just consolidate this to 
```
if (latestClockValue >= fetchedLatestClockVal) throw new Error(...)
```

Reply:
no, removing per discussion!